integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -520,4 +583,29 @@ private void callWebAppAndVerifyScaling(Domain domain, int replicas) throws Exce
       TestUtils.checkServiceCreated(podName, domainNS);
     }
   }
+
+  public static void tearDown() throws Exception {
+    StringBuffer cmd =
```

Comment:
Not a new issue.  

It's sometimes confusing that the test cleanup can occur *before* the system reports errors from a failed a test.  So the logging shows a test abruptly ending without explaining why, then a ton of cleanup logging, and only after than an error is reported...   This can get even more confusing if the tear-down itself reports problems tearing down.   I assume this is a side-effect of how jUnit works, but it turns the output of a failed test run into a bit of a jumble.    

To address this, maybe do two things:

(1) add a log message here something like:   "TEARDOWN:  Starting Test Run TearDown (cleanup and state-dump).   Note that if the test failed previous to tearDown, the error that caused the test failure may be reported after the tearDown completes.   Note that tearDown itself may report errors, but this won't affect the outcome of the test results."

(2) Minimize logging during the tear-down (unless errors occur).

Reply:
Changes are done

Reply:
Looks good!